packages/moonstone/Scroller/Scroller.js

```diff
@@ -82,6 +83,12 @@ class ScrollerBase extends Component {
 		spotlightId: PropTypes.string
 	}
 
+	constructor (props) {
+		super(props);
+
+		this.heightThreshold = ri.scale(24);
```

Comment:
Does this have issues with resizing and V8 snapshot?

Reply:
I don't think so? And even if so, shouldn't `ui/resolution` handle that case? 
FYI: `moonstone/ContextualPopupDecorator` already uses this pattern. https://github.com/enactjs/enact/blob/master/packages/moonstone/ContextualPopupDecorator/ContextualPopupDecorator.js#L246-L248

Reply:
Pretty sure that's wrong, too. :/  Maybe @JayCanuck or @Djspaceg could say?

Reply:
I believe `ri` in the constructor will result in an incorrect value in the prerender on non-default resolutions (anything non-1080p), resulting in a markup mismatch error on page load and additional re-rendering.

Reply:
The run-time cost of `ri.scale()` is pretty small; it's only multiplication except for the first invocation after a resolution change. Let's move it into `calculateScrollTop` to avoid any possible issues with resolution changes.